packaging/pre_build_script_windows.sh

```diff
@@ -0,0 +1,11 @@
+python -m pip install -U numpy packaging pyyaml setuptools wheel
+
+# Install Torch from Test Index
+python -m pip install torch==2.3.0 --index-url https://download.pytorch.org/whl/test/${CU_VERSION}
```

Comment:
This line can be complete removed for the same reason as above, since the [Install torch dependency](https://github.com/pytorch/TensorRT/actions/runs/8808671602/job/24178152262#step:8:1) step already installed the desired version.